src/test/java/uk/gov/pay/directdebit/mandate/resources/ConfirmPaymentResourceIT.java

```diff
@@ -55,7 +80,29 @@ public void confirm_shouldCreateAMandateAndUpdateCharge() {
         paymentRequestFixture.insert(testContext.getJdbi());
         transactionFixture.insert(testContext.getJdbi());
         String paymentRequestExternalId = paymentRequestFixture.getExternalId();
-        PayerFixture.aPayerFixture().withPaymentRequestId(paymentRequestFixture.getId()).insert(testContext.getJdbi());
+        PayerFixture payerFixture = PayerFixture.aPayerFixture().withPaymentRequestId(paymentRequestFixture.getId()).insert(testContext.getJdbi());
+
+        String lastTwoDigitsBankAccount = payerFixture.getAccountNumber().substring(payerFixture.getAccountNumber().length()-2);
+        String chargeDate = LocalDate.now().plusDays(4).format(DateTimeFormatter.ofPattern("dd/MM/yyyy"));
+        String emailPayloadBody = "{\"address\": \"" + payerFixture.getEmail() + "\", " +
+                "\"gateway_account_id\": \"" + gatewayAccountFixture.getExternalId() + "\"," +
```

Comment:
Probably this should be `gateway_account_external_id` as a name to be more clear, thanks